### PR TITLE
012 new role

### PR DESCRIPTION
**Pull request by orlagh-mccloskey**

### CODE REVIEW

File: app/controller/JobController.js [hunk 1]
```diff
@@ -71,4 +72,28 @@ router.get('/job-responsibility/:id', async (req, res) => {
...
+
+//take request body and response body
+//check if passed job role is valid, if not, throw exception, and render error message
+//if ok await for id form post user
+//redirect to /job-roles if not error
+//if error occurres, render error message
+router.post('/job-roles', async (req, res) => {
+    try {
+        if(AddJobRoleValidator.validateUserInput(req.body)){
+            const jobRole = JSON.parse(JSON.stringify(req.body))
```
Comment (shaunganley) [inline app/controller/JobController.js:84]: I thought just `const jobRole = req.body` would work here?

File: app/controller/JobController.js [hunk 2]
```diff
@@ -71,4 +72,28 @@ router.get('/job-responsibility/:id', async (req, res) => {
...
+            data = await JobService.addJobRole(jobRole) 
+            let success = "New job role added"
+            res.locals.success = success
+            //res.redirect('/job-roles')
+            res.render('addJobRoleView')
+            console.log("working")
```
Comment (shaunganley) [inline app/controller/JobController.js:90]: remove logging

File: app/service/JobService.js [hunk 3]
```diff
@@ -29,6 +29,32 @@ module.exports.getJobRoles = async function () {
     }
 }
 
+module.exports.addJobRole = async function (role){ 
+    try{
+        console.log(role)
```
Comment (shaunganley) [inline app/service/JobService.js:34]: remove logging

**Review by shaunganley — APPROVED** [review]
small comments

**Review by MonikaKucinska — APPROVED** [review]

**Pull request merged**